nifi-nar-bundles/nifi-standard-bundle/nifi-jolt-transform-json-ui/src/test/java/org/apache/nifi/web/standard/api/processor/TestProcessorWebUtils.java

```diff
@@ -53,7 +54,7 @@ public void testGetComponentDetailsForProcessor(){
     }
 
     @Test
-    @Ignore("Capability to get specific client revision has been removed as of commit 05a1d63090d97d06bf823821d1b0e9b5482c7cd2")
+    @Disabled("Capability to get specific client revision has been removed as of commit 05a1d63090d97d06bf823821d1b0e9b5482c7cd2")
     // FIXME java11 Capability to get specific client revision has been removed as of commit 05a1d63090d97d06bf823821d1b0e9b5482c7cd2
     //  Add test case for ProcessorWebUtils.getRequestContext() to verify proper revision is returned
     public void testGetComponentDetailsForProcessorWithSpecificClientRevision(){
```

Comment:
Instead of continuing to ignore this test method, it should be removed completely.

Reply:
Thanks for the feedback, will remove them then